installer.sh

```diff
@@ -0,0 +1,442 @@
+    if command -v sha1sum > /dev/null 2>&1; then
+        cmd_shasum="sha1sum"
+        output "  [*] sha1sum is installed" "success"
+    elif command -v shasum > /dev/null 2>&1; then
+        cmd_shasum="shasum -a 1"
+        output "  [*] shasum is installed" "success"
+    else
+        output "  [ ] No sha1sum or shasum available to verify binary" "error"
+        exit_with_error
+    fi
+}
+
+function check_directories() {
+    if [ ! -z $INSTALL_DIR ]; then
+        dir_bin="${INSTALL_DIR}"
+    elif echo $PATH | grep "$HOME/.global/bin" > /dev/null; then
+        dir_bin="$HOME/.global/bin"
+    elif echo $PATH | grep "$HOME/.local/bin" > /dev/null; then
+        dir_bin="$HOME/.local/bin"
+    fi
+
+    if ! echo $PATH | grep ${dir_bin} > /dev/null; then
+        binary="${dir_bin}/platform"
+
+        output "  [ ] ${dir_bin} is not in \$PATH.\n" "warning"
+        add_footer_note "  ⚠ The directory \"${dir_bin}\" is not in \$PATH."
+        if echo $SHELL | grep '/bin/zsh' > /dev/null
+        then
+            add_footer_note \
+                "    Run this command to add the directory to your PATH" \
+                "    echo 'export PATH=\"${dir_bin}:\$PATH\"' >> \$HOME/.zshrc"
+        elif echo $SHELL | grep '/bin/bash' > /dev/null
+        then
+            add_footer_note \
+                "    Run this command to add the directory to your PATH" \
+                "    echo 'export PATH=\"${dir_bin}:\$PATH\"' >> \$HOME/.bashrc"
+        else
+            add_footer_note \
+                "    You can add it to your PATH by adding this line at the end of your shell configuration file" \
+                "    export PATH=\"${dir_bin}:\$PATH\""
+        fi
+    else
+        output "  [*] ${dir_bin} is in \$PATH." "success"
+    fi
+
+    output "\nTarget directories" "heading"
+    output "  - Binary will be installed in ${dir_bin}"
+}
+
+function install_platformsh_homebrew() {
+    output "\nInstalling Platform.sh brew tap" "heading"
+    if ! call_user "brew tap $BREW_TAP"; then
+        output "  could not add tap to brew" "error"
+        exit_with_error
+    fi
+
+    output "\nInstalling Platform.sh CLI formula" "heading"
+
+    # running on Rosetta2?
+    arch=''
+    if [ "$(uname -m)" = "x86_64" ]; then
+        if [ "$(sysctl -in sysctl.proc_translated)" = "1" ]; then
+            arch="arch -arm64"
+        fi
+    fi
+
+    if ! call_user "$arch brew install $BREW_FORMULA"; then
+        output "  could not install Formula" "error"
+        exit_with_error
+    fi
+}
+
+function github_curl {
+    if [ -z $GITHUB_TOKEN ]; then
+        curl -fsSL $1
+        return $?
+    else
+        curl -fsSL -H "Authorization: Bearer ${GITHUB_TOKEN}" $1
+        return $?
+    fi
+}
+
+function install_platformsh_raw() {
+    # Start downloading the right version
+    output "\nDownloading the Platform.sh CLI Tool" "heading"
+
+    output "  Finding the right version for platform ${kernel}/${machine}";
```

Comment:
Nothing really happens here and the word "platform" may be confusing so this line might not be necessary

Reply:
Removed the comment completely, there's no value in there indeed.